apps/wallet-mobile/src/features/Settings/ManageCollateral/ConfirmTx/ConfirmTxScreen.tsx

```diff
@@ -51,60 +52,55 @@ export const ConfirmTxScreen = () => {
     navigateTo.failedTx()
   }
 
-  const scrollViewRef = useFlashAndScroll()
-
   if (!yoroiUnsignedTx) throw new Error('Missing yoroiUnsignedTx')
 
   return (
-    <View style={styles.root}>
-      <CurrentBalance />
-
-      <View style={{paddingTop: 16, paddingHorizontal: 16}}>
-        <Fees yoroiUnsignedTx={yoroiUnsignedTx} />
-
-        <Spacer height={4} />
-
-        <BalanceAfter yoroiUnsignedTx={yoroiUnsignedTx} />
-      </View>
-
-      <ScrollView
```

Comment:
the scroll bar flash was removed, was this intentional ? (it's inconsistent right now, not all scrolls have it) please add a task to check with Product, whether its added as default behaviour or we can drop it.

Reply:
Hi @stackchain 

This screen was copied from the send flow and if im not wrong here is not needed the scroll.

Reply:
I will confirm with product

Reply:
@stackchain 

in send flow the tx summary is dynamic